metricbeat/docs/fields.asciidoc

```diff
@@ -3193,7 +3193,7 @@ The hard limit on the number of file descriptors opened by the process. The hard
 [float]
 == cgroup Fields
 
-Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
+EXPERIMENTAL: Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
```

Comment:
@ruflin  You can use one of the following (see https://github.com/elastic/docs#experimental):
- experimental[] to get a warning box that says "This functionality is experimental and may be changed or removed completely in a future release."
- experimental[Custom text goes here] to add custom text. I'm not sure exactly how this one looks (never tried it), but give it a try!